### PR TITLE
Create pest-management-science.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/66886/style-check-pest-management-science

### CODE REVIEW

File: pest-management-science.csl [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="available at">available</term>
+    </terms>
+  </locale>
+  <macro name="type-number">
+    <group delimiter=" ">
+      <choose>
+        <if type="patent" match="none">
```
Comment (adam3smith) [inline pest-management-science.csl]: what's the purpose of this? If anything, wouldn't we expect "US Patent" to be in the genre field for patents (e.g. in Mendeley; Zotero doesn't have one)?

File: pest-management-science.csl [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+        </group>
+      </if>
+      <else>
+        <text variable="edition" prefix=", "/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="editor">
+    <names variable="editor">
+      <name initialize-with=". " delimiter=", " and="text"/>
```
Comment (adam3smith) [inline pest-management-science.csl]: in the style guide I'm seeing "ed. by" for chapters and initials as for authors. Generally the chapter format is different from what you have here & below (e.g. also not seeing the square brackets around book title, different order of editor/book title). Is that on purpose based on a published piece or an oversight?

File: pest-management-science.csl [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+          <names variable="editor"/>
+          <text macro="title"/>
+        </substitute>
+      </names>
+    </group>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="report">
+        <text variable="title" quotes="true"/>
```
Comment (adam3smith) [inline pest-management-science.csl]: are you actually seeing quotes? That seems out of character with the rest of the style.

File: pest-management-science.csl [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+    </sort>
+    <layout delimiter="," vertical-align="sup">
+      <text variable="citation-number"/>
+      <group prefix="(" suffix=")">
+        <label variable="locator" form="short" strip-periods="true"/>
+        <text variable="locator"/>
+      </group>
+    </layout>
+  </citation>
+  <bibliography et-al-min="7" et-al-use-first="6" second-field-align="flush" entry-spacing="0" hanging-indent="true">
```
Comment (adam3smith) [inline pest-management-science.csl]: It's unusual to combine hanging-indent and second-field-align and I'm not sure it'll work well with most processors. Was that on purpose?

File: pest-management-science.csl [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+          <text variable="publisher-place"/>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if type="webpage post-weblog">
+        <group delimiter=": ">
+          <text term="available at" text-case="capitalize-first"/>
```
Comment (adam3smith) [inline pest-management-science.csl]: the available term isn't in the author guidelines?

File: pest-management-science.csl [hunk 6]
```diff
@@ -0,0 +1,251 @@
...
+      <date-part name="year"/>
+    </date>
+  </macro>
+  <macro name="issued-year">
+    <date variable="issued" prefix=" (" suffix=")">
+      <date-part name="year"/>
+    </date>
+  </macro>
+  <macro name="accessed">
+    <date variable="accessed">
```
Comment (adam3smith) [inline pest-management-science.csl]: author guidelines do have an "accessed" term here

File: pest-management-science.csl (hunk 1 above)
Comment (POBrien333) [inline pest-management-science.csl]: No clue. that's from the template.

File: pest-management-science.csl (hunk 2 above)
Comment (POBrien333) [inline pest-management-science.csl]: oversight. used a publication as a base and forgot to double check chapters. thanks for spotting that.

File: pest-management-science.csl (hunk 5 above)
Comment (POBrien333) [inline pest-management-science.csl]: again. based it on a publication. thanks.

File: pest-management-science.csl (hunk 4 above)
Comment (POBrien333) [inline pest-management-science.csl]: that's how the papers look in the pdfs. which one should I delete?

File: pest-management-science.csl (hunk 2 above)
Comment (rmzelle) [inline pest-management-science.csl]: > used a publication as a base

Per @adam3smith, can you link to a publication that shows this?

File: pest-management-science.csl (hunk 5 above)
Comment (rmzelle) [inline pest-management-science.csl]: > again. based it on a publication.

Idem.

**Comment by rmzelle** [comment]
(style metadata looks good to me)

File: pest-management-science.csl (hunk 4 above)
Comment (POBrien333) [inline pest-management-science.csl]: pinging Dr @adam3smith  :)

**Comment by adam3smith** [comment]
Thanks -- I left the second-field-align and hanging indent in: looks like that'll either work (html) or not do any harm (RTF)

**Pull request merged**